src/stravalib/model.py

```diff
@@ -889,22 +888,65 @@ class SegmentEffort(BaseEffort):
     achievements: Optional[list[SegmentEffortAchievement]] = None
 
 
+class MetaActivity(strava_model.MetaActivity, BoundClientEntity):
+    @lazy_property
+    def comments(self) -> BatchedResultsIterator[ActivityComment]:
+        """Retrieves comments for a specific activity id."""
+        assert self.bound_client is not None
+        return self.bound_client.get_activity_comments(self.id)
```

Comment:
All these lazy properties can be defined at the meta level, because they only depend on the `id` field.